packages/services/api/src/modules/organization/providers/organization-manager.ts

```diff
@@ -336,15 +338,25 @@ export class OrganizationManager {
     });
 
     if (result.ok) {
-      await this.activityManager.create({
-        type: 'ORGANIZATION_CREATED',
-        selector: {
-          organizationId: result.organization.id,
-        },
-        user,
-      });
+      await Promise.all([
+        this.activityManager.create({
+          type: 'ORGANIZATION_CREATED',
+          selector: {
+            organizationId: result.organization.id,
+          },
+          user,
+        }),
+      ]);
     }
 
+    await this.auditLog.record({
+      eventType: 'ORGANIZATION_CREATED',
+      organizationId: result.ok ? result.organization.id : slug,
+      metadata: {
+        organizationSlug: slug,
+      },
+    });
```

Comment:
@TuvalSimha This should only be inserted if result.ok exists. Why did you choose to insert the slug instead of the organization id if the organizationId does not exist?